### PR TITLE
create index.md for next small scale meeting

### PR DESCRIPTION
**Pull request by hrhotz**
WIP - not ready to merge

### CODE REVIEW

**Comment by hrhotz** [comment]
now, it is ready to be merged - thank you

File: content/events/2022-09-small-scale/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+contact: "Lucille Delisle"
+tags: []
+subsites: [global, us]
+---
+
+
+
+Please <a href="https://epfl.zoom.us/j/62244337954?pwd=L2ZVSytTdlpaM1pvbjZpWjNjUm9JZz09">join us</a> 
+at 4 PM CEST (<a href="https://www.timeanddate.com/worldclock/fixedtime.html?msg=Small+Scale+Galaxy+Admins+Meeting&iso=20220920T16&p1=1229&ah=1">see in your timezone</a>) 
+for a discussion about 'How I switched from an old galaxy to an ansible galaxy?' lead by Lucille Delisle.
```
Comment (beatrizserrano) [inline content/events/2022-09-small-scale/index.md:21]:
```suggestion
for a discussion about 'How I switched from an old Galaxy to an ansible Galaxy?' lead by Lucille Delisle. 
```

File: content/events/2022-09-small-scale/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+gtn: false
+contact: "Lucille Delisle"
+tags: []
+subsites: [global, us]
+---
+
+
+
+Please <a href="https://epfl.zoom.us/j/62244337954?pwd=L2ZVSytTdlpaM1pvbjZpWjNjUm9JZz09">join us</a> 
+at 4 PM CEST (<a href="https://www.timeanddate.com/worldclock/fixedtime.html?msg=Small+Scale+Galaxy+Admins+Meeting&iso=20220920T16&p1=1229&ah=1">see in your timezone</a>)
```
Comment (beatrizserrano) [inline content/events/2022-09-small-scale/index.md:20]:
```suggestion
Please [join us](https://epfl.zoom.us/j/62244337954?pwd=L2ZVSytTdlpaM1pvbjZpWjNjUm9JZz09) at 4 PM CEST ([see in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Small+Scale+Galaxy+Admins+Meeting&iso=20220920T16&p1=1229&ah=1)) 
```

**Review by beatrizserrano — APPROVED** [review]
Danke sehr!

**Pull request merged**